plane/source/docs/advanced-failsafe-configuration.rst

```diff
@@ -262,6 +262,13 @@ the aircraft is a danger to people or other aircraft. To force an
 immediate termination you should use the :ref:`AFS_TERMINATE<AFS_TERMINATE>` parameter. By
 setting that parameter to 1 the aircraft will immediately terminate.
 
+AFS_OPTIONS
+------------------
+In addition, the behavior of AFS can be modified by the setting of the :ref:`AFS_OPTIONS<AFS_OPTIONS>` bitmask parameter (no bits are set by default):
+
+- bit 0 (+1), if set, the aircraft will continue the mission even after data link is recovered. If not set, the aircraft will return to the mission item right before GCS failsafe occurs 
```

Comment:
"continue the mission" ?doesnt this mean "continue the mission sequence jumped to by the AFS"?
what is data link? RC?GCS? either?

Reply:
Not quite, continuing the mission means it will keep flying with the current mission item. If set, there would be no jumping. I reword it slightly. I hope it's a bit better.
 
The phrase data link has been changed.